core/src/main/java/io/questdb/cairo/map/FastMap.java

```diff
@@ -94,19 +105,33 @@ public FastMap(
             double loadFactor,
             HashFunction hashFunction,
             int maxResizes
+    ) {
+        this(pageSize, keyTypes, valueTypes, keyCapacity, loadFactor, hashFunction, maxResizes, -1);
```

Comment:
nit: we could pass actual tag we want and remove conditionals from the main constructor